### ISSUE TITLE
Unclear documentation crossref

### ISSUE THREAD

**Issue by dmenne** [quarto-dev/quarto-web#493]
It is not clear in the documentation on which level/sublevel to put crossref. I got errors in any position I tried

      crossref:
        appendix-title: "App."
        appendix-delim: ":"

**Comment by dragonstyle** [comment]
Most of the examples in the documentation show usage at the root level of document front matter. Is this where you are trying to use it? If you can provide an example of that you're trying that isn't working, we can try to help get to the bottom of why the errors are occurring and potentially clean up our documentation at the same time...

**Comment by dmenne** [comment]
![2022-11-11 15_04_03-mybook - RStudio](https://user-images.githubusercontent.com/506275/201356319-d4b62d16-6070-4d1f-ad72-a4add3501af5.png)




```
ERROR: Project _quarto.yml validation failed.

In file C:\tmp\mybook\_quarto.yml
(line 19, columns 3--17) property name appendix-title is invalid
18: crossref:
19:   appendix-title: "App."
     ~~~~~~~~~~~~~~~
20:   appendix-delim: ":"
i The error happened in location crossref:appendix-title.


In file C:\tmp\mybook\_quarto.yml
(line 20, columns 3--17) property name appendix-delim is invalid
19:   appendix-title: "App."
20:   appendix-delim: ":"
     ~~~~~~~~~~~~~~~
i The error happened in location crossref:appendix-delim.

**Comment by dragonstyle** [comment]
Ahh thank you for sharing that- the issue is actually with our validation (which has a bug that it fails to validate these properly). We've fixed that for our 1.2 release, which is available as a release candidate here:

https://quarto.org/docs/download/
